src/components/Cards/MeetingCard.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Card } from "./";
+import colors from "../../styles/colors";
+import { fontSizes } from "../../styles/fonts";
+
+type MeetingCardProps = {
+  imgSrc: string;
+  imgOverlayText: string;
+  committee: string;
+  tags: [string];
+  actions: string;
+  excerpt?: string;
+};
+
+const HeadingSection = styled.div`
+  margin: 0 0 2rem 0;
+`;
+
+const SectionTitle = styled.div`
+  padding-bottom: 0.4rem;
+  font-size: ${fontSizes.font_size_3};
+  color: ${colors.grey};
+`;
+
+const HeadingContent = styled.div`
+  font-size: ${fontSizes.font_size_5};
+  font-weight: bold;
+  color: ${colors.black};
+`;
+
+const BodySection = styled.div`
+  margin: 0 0 1rem 0;
+`;
+
+const SectionContent = styled.div`
+  font-size: ${fontSizes.font_size_4};
+  font-weight: bold;
+  color: ${colors.black};
+`;
+
+const SearchExcerpt = styled.div`
+  font-size: ${fontSizes.font_size_3};
+  color: ${colors.grey};
+`;
+
+export const MeetingCard = (props: MeetingCardProps) => {
```

Comment:
It is not needed to typed MeetingCard with `FunctionComponent<MeetingCardProps>` as long as the props is typed?

Reply:
If I understand your question correctly, yes! The component gets its type from the props:
![Screen Shot 2020-09-19 at 3 42 23 PM](https://user-images.githubusercontent.com/13170966/93690578-ca9fed00-fa8e-11ea-9dfa-4ec44b3b6c98.png)